README.md

```diff
@@ -182,10 +180,10 @@ $promises = array(
     $client->imageInspect('ubuntu'),
 );
 
-$inspections = Block\awaitAll($promises, Loop::get());
+$inspections = \React\Async\await(\React\Async\parallel($promises));
```

Comment:
Can you add a use statement in this example as we did in [clue/reactphp-mq#34](https://github.com/clue/reactphp-mq/pull/34/files).